### PR TITLE
feat: support numpad enter on the input component

### PR DESCRIPTION
**Pull request by ShaneMaglangit**
Support submission of Input changes using the Numpad Enter key.

**Changes**

- [x] Add check for "NumpadEnter" on SidePanel/Input.tsx key down listener.

### CODE REVIEW

**Comment by subhoghoshX** [comment]
Yeah noticed it, I'm sure this PR #41 caused it. Btw noticed you might not be using Prettier. The spacing doesn't look consistent.

**Comment by jalezi** [comment]
> Yeah noticed it, I'm sure this PR #41 caused it. Btw noticed you might not be using Prettier. The spacing doesn't look consistent.

Yeah, my bad. Maybe it's time to add some [tests](https://github.com/subhoghoshX/laureate/discussions/39#discussion-4447603).

**Comment by ShaneMaglangit** [comment]
> Yeah noticed it, I'm sure this PR #41 caused it. Btw noticed you might not be using Prettier. The spacing doesn't look consistent.

I was actually uncertain how to indented two cases together. Co-pilot generated that nested structure so I just went along with it.

File: components/SidePanel/Input.tsx [hunk 1]
```diff
@@ -23,8 +23,9 @@ export default function Input({ data, action: setData, label }: Props) {
         setData(() => +dataBuffer - 1);
         break;
       case "Enter":
-        setDataOrResetDataBuffer();
-        break;
+        case "NumpadEnter":
+          setDataOrResetDataBuffer();
+          break;
```
Comment (subhoghoshX) [inline components/SidePanel/Input.tsx:28]:
```suggestion
      case "NumpadEnter":
        setDataOrResetDataBuffer();
        break;
```

**Review by subhoghoshX — APPROVED** [review]

**Pull request merged**